bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/sipclient.js

```diff
@@ -0,0 +1,15 @@
+import { pt, pb } from '../helpers.js'
+
+export default () => [
+  pt('height', 'Component height', 'Height of the whole component in pixels'),
+  pt('websocketUrl', 'Websocket URL', 'URL of the WebRTC SIP websocket, e.g. \'wss://siphost:8089/ws\'. For Android & iOS, you need wss (WebSocket secured).'),
+  pt('domain', 'Domain', 'SIP Domain'),
+  pt('username', 'Username', 'SIP Username'),
+  pt('password', 'Password', 'SIP Password'),
+  pt('sipAddress', 'SIP Address', 'SIP Address (phone number) to call'),
+  pb('enableTones', 'Enable tones', 'Enable ringback and ring tone. Might cause issues with your browser, so that your call fails.'),
+  pb('hideCallerId', 'Hide caller id', 'Hides the username of the remote party on incoming call'),
+  pt('phonebook', 'Phonebook', 'Comma-separated list of \'value=label\' used to display names for incoming calls. Required for intercom mode.'),
+  pb('intercomEnabled', 'Enable Intercom Mode', 'If enabled, you can make outgoing calls to more than one SIP user using the phonebook. ' +
```

Comment:
So i'm thinking we should get rid of the `intercomEnabled`, as well as the `sipAddress` option  and just have the `phonebook` option instead.  All of these are rather redundant, and really it's not enabling an intercom as much as its supporting dialing more than one number.  My suggestion for the logic is that if you have one number in the phonebook, it dials that without prompting (same as having the `sipAddress` option), if you have more then one number then it prompts for which number to dial (same as having `intercom` option) , and if no numbers are configured it alerts you to add a number to the phone book, or we simply hide the dial button.   This achieves the same functionality, but with 1 option instead of 3.

Reply:
That's a very good idea! I will surely implement that, thanks.

You are right, the configuration that is enabling the intercom functionality is setting up the local device configuration and not that `intercomEnabled` option.

Reply:
Awesome, and yeah the  local configuration is a great idea, not exactly an "intercom" as much as its setting the device specific SIP extension , which is pretty fundamental for anything other then trivial use cases.  For my use case, i have several android tablets around the house that have their own SIP extension using a custom version of linphone i have modified, so this might be a good alternative. 

Just FYI, In the VOIP world a true "intercom" functionality uses multicast to stream a common RTP audio stream to all phones on the network, who then play the audio without the user having to do anything, at that point a user can answer or push a button to talk which either multicasts back the audio, or creates a direct SIP call to that device.  Might be fun to look at in the future.  There are also other implementations that use a central mixer/bridge (like Freeswitch or Asterisk) , but those tend to not be standard.

Reply:
Thanks for the intercom explanation.

@hmerk and I use Intercom as word for supporting communication between multiple openHAB WebUIs with the local config.

Reply:
@florian-h05 interesting how you assume what I understand as intercom.
In fact, as a start, it would be sufficient how you see it, but I rally would like to see a true intercom like @digitaldan describes it from the real VoIP world. But this can be addressed later as an enhancement.

Reply:
@hmerk 
Seems to be a misunderstanding on my part.

Reply:
No problem at all😉

Reply:
and to be fair, sometimes what i am talking about is called a "paging" intercom, so its very confusing and used a lot in different contexts depending on who you talk to :-)

One other option i was thinking about is a feature called "early media" which is supported in JSSIP, this essentially allows the receivers to accept video and audio during the connection phase instead of hearing the ring back sounds.  In this regard, you kinda get the same result as a paging intercom, although the client would need to call multiple extensions in parallel unless a proxy server sits in the middle and does that on its own (which is what i have at home) .  This is used a lot for example in door or gate phones, where you want to see the remote video before you answer the call.

Reply:
Done, see https://github.com/openhab/openhab-webui/pull/1453/commits/144b953403eb10fea2e1dcd5959474b2e4db1aed